pkg/agent/manager/manager.go

```diff
@@ -145,12 +153,14 @@ func (m *manager) Err() error {
 func (m *manager) run() error {
 	m.setRunning(true)
 	m.t.Go(m.synchronizer)
-	m.t.Go(m.rotator)
+	m.t.Go(m.startSVIDObserver)
+	m.t.Go(m.startBundleObserver)
+	m.svid.Start()
 	return nil
 }
 
 func (m *manager) synchronizer() error {
-	t := time.NewTicker(m.syncFreq)
+	t := time.NewTicker(m.c.SyncInterval)
 
```

Comment:
not related to this change, but this ticker needs to be `Stop()`'d otherwise it will leak a goroutine.

Reply:
:+1: